vspace/src/main/java/edu/asu/diging/vspace/web/publicview/ExhibitionModuleController.java

```diff
@@ -23,9 +23,11 @@
     @Autowired
     private ISpaceManager spaceManager;
 
-    @RequestMapping(value = "/exhibit/{spaceId}/module/{id}")
-    public String module(@PathVariable("id") String id, @PathVariable("spaceId") String spaceId, Model model)
+    @RequestMapping(value = { "/exhibit/{spaceId}/module/{id}", "/preview/{previewId}/{spaceId}/module/{id}" })
```

Comment:
The constant should also be used within the requestmapping annotation (in all controllers). I'm fairly certain that you can do that.